format/format.go

```diff
@@ -170,6 +170,36 @@ func formatTableColumn(dst io.Writer, col model.TableColumn) error {
 		return err
 	}
 
+	if col.HasEnumValues() {
```

Comment:
Can `col` have *both* Enum values and Set values? if so, wouldn't it create a text like `(a, b, c) (d, e, f)`?

Maybe we should allow setting one of enum or set. then this clause can be an `if ... else` instead of `if .. if `

Reply:
i agree